modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java

```diff
@@ -19,4 +34,108 @@
 )
 public class SavedContentEntryLocalServiceImpl
 	extends SavedContentEntryLocalServiceBaseImpl {
+
+	@Override
+	public void addEntryResources(
+			SavedContentEntry entry, boolean addGroupPermissions,
+			boolean addGuestPermissions)
+		throws PortalException {
+
+		_resourceLocalService.addResources(
+			entry.getCompanyId(), entry.getGroupId(), entry.getUserId(),
+			SavedContentEntry.class.getName(), entry.getSavedContentEntryId(),
+			false, addGroupPermissions, addGuestPermissions);
+	}
```

Comment:
I could be mistaken, but it looks like these `addEntryResources` methods are not called from the outside. Can you make the private/protected?

Reply:
it's only used here so I removed the methods and inlined it.
if on next moves are needed we can do it.